lib/index.js

```diff
@@ -192,27 +153,13 @@ export default {
           return [];
         }
 
-        // Construct the list of linters to be passed to the CLI by looking at
-        // the linters made available for our language grammar by the LinterMap,
-        // and whether or not the engines are enabled in a user's config file.
-        if (Object.prototype.hasOwnProperty.call(linterMap, grammarName) === true) {
-          // for (const linter of linterMap[grammarName]) {
-          linterMap[grammarName].forEach((linter) => {
-            linterNames.add(linter);
-          });
-        }
-        const configEnabledEngines = getEnabledEngines(configurationFilePath);
-        const linterEnabledEngines = Array.from(linterNames).filter(
-          linterName => configEnabledEngines.has(linterName));
-
         // Construct the command line invocation which runs the Code Climate CLI
         const relpath = atom.project.relativizePath(filePath).pop();
-        const execArgs = ['analyze', '-f', 'json'].concat(
-          getEngineArgs(linterEnabledEngines),
-          [relpath, '</dev/null']);
+        const execArgs = ['analyze', '-f', 'json', relpath];
         const execOpts = {
           cwd: dirname(configurationFilePath),
           uniqueKey: `linter-codeclimate::${relpath}`,
+          env: process.env,
```

Comment:
I think the CI error does not have to do with the env variables, but the new wrapper for the codeclimate cli. I've been testing it with v0.62.0 and my PR #65 works flawlessly. If I use the recent v0.63.0 or v0.63.1 ones, it fails.

The tests are failing because, as @Arcanemagus says, codeclimate command is failing. If you run the original command `codeclimate analyze -f json -e fixme -e rubocop cool_code.rb </dev/null` you get this output:

```shell
Error: ! analyze
! -f
! json
! -e
! fixme
! -e
! rubocop
! cool_code.rb
! </dev/null
Please supply only one file for analysis
```

If you remove the `</dev/null` argument, then you get this output:

```shell
Error: ! analyze
! -f
! json
! -e
! fixme
! -e
! rubocop
! cool_code.rb
[DEBUG] Is a directory @ io_fillbuf - fd:7 /config.yml
error: (Errno::ENOENT) No such file or directory @ rb_sysopen - /code/.codeclimate.yml
[DEBUG] backtrace: /usr/src/app/lib/cc/analyzer/filesystem.rb:19:in `read'
	/usr/src/app/lib/cc/analyzer/filesystem.rb:19:in `read_path'
	/usr/src/app/lib/cc/cli/analyze.rb:72:in `config'
	/usr/src/app/lib/cc/cli/analyze.rb:76:in `apply_config_options'
	/usr/src/app/lib/cc/cli/analyze.rb:23:in `initialize'
	/usr/src/app/lib/cc/cli/runner.rb:25:in `new'
	/usr/src/app/lib/cc/cli/runner.rb:25:in `run'
	/usr/src/app/lib/cc/cli/runner.rb:9:in `run'
	/usr/src/app/bin/codeclimate:6:in `<main>'
```

I think it would be useful to add the env var `"env":{"CODECLIMATE_DEBUG":1}` if `atom.inDevMode()` to the command options, to attach a more detailed info about the crash to the CI reports and local tests.

The codeclimate command fails, so it returns null, and the tests fail because they expect a JSON object to be returned, but they get `null` instead, and thus cannot access the property '0'.